test/tornado_tests/test_motor_collection.py

```diff
@@ -261,6 +262,25 @@ def test_sub_collection(self):
             self.assertEqual(read_preference, c.read_preference)
             self.assertEqual(codec_options, c.codec_options)
 
+    @gen_test
+    async def test_async_create_encrypted_collection(self):
```

Comment:
This test is failing because it needs to be skipped unless pymongo>=4.4 is installed.

Reply:
Done.